lib/ui/button.tsx

```diff
@@ -29,7 +29,7 @@ const buttonVariants = cva(
         neutral:
           "bg-f1-background-secondary text-f1-foreground hover:bg-f1-background-secondary-hover",
         critical:
-          "border border-solid border-f1-border bg-f1-background-secondary text-f1-foreground-critical hover:border-none hover:bg-f1-background-critical-bold hover:text-f1-foreground-inverse",
+          "border border-solid border-f1-border bg-f1-background-secondary text-f1-foreground-critical hover:border-transparent hover:bg-f1-background-critical-bold hover:text-f1-foreground-inverse",
```

Comment:
Small fix to the critical button. We need to set _some_ border, as setting a border-none makes it shrink.

Reply:
That's the usual fix for this situation 👍